src/screens/Products/ProductDetail.js

```diff
@@ -0,0 +1,266 @@
+                  this.setState({ editExpanded: !this.state.editExpanded })
+                }
+              />
+            </CardSection>
+            <CardSection>
+              <ProfilePicture
+                source={{ uri: this.props.mainPictureURL }}
+                style={styles.image}
+              />
+            </CardSection>
+            {this.renderEditButton()}
+          </Card>
+          <Card>
+            <CardSection
+              style={{
+                backgroundColor: 'rgb(71, 77, 84)',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.headerText}>{this.props.MainCategory}</Text>
+            </CardSection>
+            <CardSection
+              style={{
+                backgroundColor: 'white',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.descriptionStyle}>
+                {this.props.Description}
+              </Text>
+            </CardSection>
+          </Card>
+          <Card>
+            <CardSection
+              style={{
+                backgroundColor: 'rgb(71, 77, 84)',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.headerText}>Info</Text>
+            </CardSection>
+            <CardSection
+              style={{
+                backgroundColor: 'white',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.ProductInfo}>Sub-Cat</Text>
+              <Text style={styles.SectionListItemS}>
+                {this.props.SubCategory}
+              </Text>
+            </CardSection>
+            <CardSection
+              style={{
+                backgroundColor: 'white',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.ProductInfo}>Price</Text>
+              <Text style={styles.SectionListItemS}>
+                {'$' + this.props.StandardPrice}
+              </Text>
+            </CardSection>
+            <CardSection
+              style={{
+                backgroundColor: 'white',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.ProductInfo}>In Stock</Text>
+              <Text style={styles.SectionListItemS}>{this.props.Quantity}</Text>
+            </CardSection>
+            <CardSection
+              style={{
+                backgroundColor: 'white',
+                justifyContent: 'space-between'
+              }}
+            >
+              <Text style={styles.ProductInfo}>Gender</Text>
+              <Text style={styles.SectionListItemS}>{this.props.Gender}</Text>
+            </CardSection>
+            <CardSection
+              style={{
+                backgroundColor: 'white',
+                justifyContent: 'space-between'
+              }}
+            >
```

Comment:
Lines 111 to 165 I believe is all repeated code so it might be worth making a list of CardSections here, or perhaps another component that does this rendering for you.